doc/Type/Map.pod6

```diff
@@ -44,7 +44,7 @@ a C<Map> has been initialized:
    $m{ 'c' } = 'foo'; # WRONG!
                       # Cannot modify an immutable Str
 
-Unlike its mutable companion type L<Hash>, Map cannot be parameterized by key or value types.
+Unlike its mutable companion type L<Hash|/type/Hash>, Map cannot be parameterized by key or value types.
```

Comment:
This is fine, but while you're at it, maybe say *a* Map or Map*s*?

Reply:
Right!